.github/workflows/pckdown.yaml

```
@@ -14,114 +12,54 @@ jobs:
           branch: gh-pages
           folder: docs
-          target-folder: branch/${{ github.head_ref }}
+          target-folder: ${{ github.event_name == 'pull_request' && format('branch/{0}', github.head_ref) || '' }}
 
       - name: Comment PR
         if: github.event_name == 'pull_request' && github.event.action == 'opened'
         uses: actions/github-script@v7
-        env:
-          HEAD_REF: ${{ github.head_ref }}
         with:
           script: |
```

actionlint (1.7.9)

"github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)